lib/token-manager.js

```diff
@@ -0,0 +1,13 @@
+const db = require('./db')
+
+function getTokenList () {
+  const sql = `select * from user_tokens`
+  return db.$any(sql)
+}
+
+function revokeToken (token) {
+  const sql = `delete from user_tokens where token = $1`
+  return db.$none(sql, [token])
+}
+
+module.exports = { getTokenList, revokeToken }
```

Comment:
This file is removed in dev after the authentication merge, most likely reappeared on a rebase or something. You can remove it